pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -381,7 +381,7 @@ func checkCrcDnsmasqConfigFile() (bool, error) {
 	if err != nil {
 		return false, fmt.Errorf("File not found: %s: %s", crcDnsmasqConfigPath, err.Error())
 	}
-	config, err := ioutil.ReadFile(crcDnsmasqConfigPath)
+	config, err := ioutil.ReadFile(filepath.Clean(crcDnsmasqConfigPath))
```

Comment:
Here it would be a false positive as crcDnsmasqConfigPath is the output of filepath.Join(constantDirname, constantFilename)

Reply:
yes, so either we just don't put `filepath.Clean` and put a `nosec` comment. I preferred to have this.